packages/core/src/connectors/mock.ts

```diff
@@ -28,35 +38,21 @@ export type MockConnectorOptions = {
-};
-
 export class MockConnector extends Connector {
   private _accounts: MockConnectorAccounts;
   private _accountIndex = 0;
-  private _options: MockConnectorOptions;
   private _connected = false;
   private _chainId: bigint = devnet.id;
+  public _options: MockConnectorOptions;
```

Comment:
Should not have a leading `_` if public.

```suggestion
  public options: MockConnectorOptions;
```